Formula/s/selene-cli.rb

```diff
@@ -0,0 +1,26 @@
+class SeleneCli < Formula
+  desc "Formatting technology for Swift source code"
+  homepage "https://github.com/zunda-pixel/selene"
+  url "https://github.com/zunda-pixel/selene.git",
+      tag:      "1.1.1",
+      revision: "703adbc3fced409da7ecf20ea2b7e162ff6f4986"
+  license "Apache-2.0"
+  version_scheme 1
+  head "https://github.com/zunda-pixel/selene.git", branch: "main"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+  
+  uses_from_macos "swift"
+
+  def install
+    system "swift", "build", "--disable-sandbox", "-c", "release"
+    bin.install ".build/release/selene"
+  end
+  
+  test do
+    system "false"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
I added Test